.github/workflows/update_package_properties.yml

```diff
@@ -0,0 +1,50 @@
+name: Update Package Properties
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PRJ_NAME: "citus"
+  TAG_NAME: ${{ github.event.inputs.tag_name }}
+  FANCY: ${{ github.event.inputs.fancy }}
+  FANCY_VERSION_NO: ${{ github.event.inputs.fancy_version_no }}
+  MICROSOFT_EMAIL: ${{ github.event.inputs.microsoft_email }}
+  NAME: ${{ github.event.inputs.name }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: 'The tag name on the project repo to be released'
+      fancy:
+        description: 'If fancy versioning is required set it "True"'
+        default: 'False'
+      fancy_version_no:
+        description: 'If fancy versioning is True, then it should be set e.g 1'
+        default: '1'
+      microsoft_email:
+        description: 'Email to be written on changelogs'
+        default: 'gindibay@microsoft.com'
+      name:
+        description: 'Name to be written on changelogs'
+        default: 'Gurkan Indibay'
+jobs:
+  update_package_properties:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: sudo apt install libcurl4-openssl-dev libssl-dev
+
+      - name: Clone Tools branch
+        run: git clone --branch tools-transition https://github.com/citusdata/tools.git
```

Comment:
We can add `--depth=1` for a faster operation

Also how about using `develop` for now, and starting to use tags after we are done with the whole transition?

Reply:
Fixed using 0.8.0